kubernetes/samples/scripts/domain-lifecycle/helper.sh

```diff
@@ -0,0 +1,662 @@
+# !/bin/sh
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+#
+# Function to get server start policy at cluster level
+# $1 - Domain resource in json format
+# $2 - Name of cluster
+# $3 - Return value for cluster level server start policy.
+#      Legal return values are "NEVER" or "IF_NEEDED" or "".
+#
+function getClusterPolicy {
+  local domainJson=$1
+  local clusterName=$2
+  local __clusterPolicy=$3
+  local effectivePolicy=""
+
+  clusterPolicyCmd="(.spec.clusters[] \
+    | select (.clusterName == \"${clusterName}\")).serverStartPolicy"
+  effectivePolicy=$(echo ${domainJson} | jq "${clusterPolicyCmd}")
+  if [ "${effectivePolicy}" == "null" ]; then
+    effectivePolicy=""
+  fi
+  eval $__clusterPolicy=${effectivePolicy}
+}
+
+#
+# Function to get server start policy at domain level
+# $1 - Domain resource in json format
+# $2 - Return value containing domain level server start policy.
+#      Legal retrun values are "NEVER" or "IF_NEEDED" or "ADMIN_ONLY".
+#
+function getDomainPolicy {
+  local domainJson=$1
+  local __domainPolicy=$2
+  local effectivePolicy=""
+
+  eval $__domainPolicy="IF_NEEDED"
+  domainPolicyCommand=".spec.serverStartPolicy"
+  effectivePolicy=$(echo ${domainJson} | jq "${domainPolicyCommand}")
+  if [ "${effectivePolicy}" == "null" ]; then
```

Comment:
If you add `|| "${effectivePolicy}" == ""` then any callers of this method won't have to check for the "" case...